### PR TITLE
feat(content,port): Add mutated forest creatures

### PR DESCRIPTION
**Pull request by Zlorthishen**
## Describe the solution
 - https://github.com/CleverRaven/Cataclysm-DDA/pull/45065

## Additional context
does not port over the `iridescent cat`, what will be another PR
image

### CODE REVIEW

**Comment by chaosvolt** [comment]
Eheheheh, Cerbearus

**Comment by Lamandus** [comment]
grafik
grafik
grafik
looks fine as is, but we also have "mutant organs" I am not sure if other mutants in our game are using regular offal or the mutated version. If not mutated, then it is fine.

File: data/json/harvest.json [hunk 1]
```diff
@@ -218,6 +218,22 @@
...
+  {
+    "id": "mutant_mammal_small_fur",
+    "//": "does not drop stomach or other small organs",
+    "type": "harvest",
+    "entries": [
+      { "drop": "mutant_meat", "type": "flesh", "mass_ratio": 0.28 },
+      { "drop": "mutant_meat_scrap", "type": "flesh", "mass_ratio": 0.05 },
+      { "drop": "lung", "type": "flesh", "mass_ratio": 0.0035 },
+      { "drop": "liver", "type": "offal", "mass_ratio": 0.01 },
+      { "drop": "brain", "type": "flesh", "mass_ratio": 0.005 },
```
Comment (chaosvolt) [inline data/json/harvest.json:230]:
```suggestion
      { "drop": "brain", "type": "flesh", "mass_ratio": 0.005 },
      { "drop": "mutant_offal", "type": "offal", "mass_ratio": 0.01 },
```

File: data/json/harvest.json [hunk 2]
```diff
@@ -237,6 +253,25 @@
...
+    "type": "harvest",
+    "entries": [
+      { "drop": "mutant_meat", "type": "flesh", "mass_ratio": 0.3 },
+      { "drop": "mutant_meat_scrap", "type": "flesh", "mass_ratio": 0.03 },
+      { "drop": "lung", "type": "flesh", "mass_ratio": 0.0035 },
+      { "drop": "liver", "type": "offal", "mass_ratio": 0.01 },
+      { "drop": "brain", "type": "flesh", "mass_ratio": 0.005 },
+      { "drop": "sweetbread", "type": "flesh", "mass_ratio": 0.002 },
+      { "drop": "kidney", "type": "offal", "mass_ratio": 0.002 },
+      { "drop": "stomach", "scale_num": [ 1, 1 ], "max": 1, "type": "offal" },
```
Comment (chaosvolt) [inline data/json/harvest.json:268]:
```suggestion
      { "drop": "brain", "type": "flesh", "mass_ratio": 0.005 },
      { "drop": "mutant_offal", "type": "offal", "mass_ratio": 0.01 },
      { "drop": "stomach", "scale_num": [ 1, 1 ], "max": 1, "type": "offal" },
```

File: data/json/harvest.json [hunk 3]
```diff
@@ -256,6 +291,44 @@
...
+    "type": "harvest",
+    "entries": [
+      { "drop": "meat", "type": "flesh", "mass_ratio": 0.3 },
+      { "drop": "meat_scrap", "type": "flesh", "mass_ratio": 0.03 },
+      { "drop": "lung", "type": "flesh", "mass_ratio": 0.0035 },
+      { "drop": "liver", "type": "offal", "mass_ratio": 0.01 },
+      { "drop": "brain", "type": "flesh", "mass_ratio": 0.005 },
+      { "drop": "sweetbread", "type": "flesh", "mass_ratio": 0.002 },
+      { "drop": "kidney", "type": "offal", "mass_ratio": 0.002 },
+      { "drop": "stomach", "scale_num": [ 1, 1 ], "max": 1, "type": "offal" },
```
Comment (chaosvolt) [inline data/json/harvest.json:306]:
```suggestion
      { "drop": "brain", "type": "flesh", "mass_ratio": 0.005 },
      { "drop": "mutant_offal", "type": "offal", "mass_ratio": 0.01 },
      { "drop": "stomach", "scale_num": [ 1, 1 ], "max": 1, "type": "offal" },
```

File: data/json/harvest.json [hunk 4]
```diff
@@ -256,6 +291,44 @@
...
+    "type": "harvest",
+    "entries": [
+      { "drop": "mutant_meat", "type": "flesh", "mass_ratio": 0.3 },
+      { "drop": "mutant_meat_scrap", "type": "flesh", "mass_ratio": 0.03 },
+      { "drop": "lung", "type": "flesh", "mass_ratio": 0.0035 },
+      { "drop": "liver", "type": "offal", "mass_ratio": 0.01 },
+      { "drop": "brain", "type": "flesh", "mass_ratio": 0.05 },
+      { "drop": "sweetbread", "type": "flesh", "mass_ratio": 0.002 },
+      { "drop": "kidney", "type": "offal", "mass_ratio": 0.002 },
+      { "drop": "stomach", "scale_num": [ 1, 1 ], "max": 1, "type": "offal" },
```
Comment (chaosvolt) [inline data/json/harvest.json:325]:
```suggestion
      { "drop": "brain", "type": "flesh", "mass_ratio": 0.005 },
      { "drop": "mutant_offal", "type": "offal", "mass_ratio": 0.01 },
      { "drop": "stomach", "scale_num": [ 1, 1 ], "max": 1, "type": "offal" },
```

File: data/json/harvest.json [hunk 5]
```diff
@@ -326,6 +399,25 @@
...
+    "type": "harvest",
+    "entries": [
+      { "drop": "mutant_meat", "type": "flesh", "mass_ratio": 0.32 },
+      { "drop": "mutant_meat_scrap", "type": "flesh", "mass_ratio": 0.01 },
+      { "drop": "lung", "type": "flesh", "mass_ratio": 0.0035 },
+      { "drop": "liver", "type": "offal", "mass_ratio": 0.01 },
+      { "drop": "brain", "type": "flesh", "mass_ratio": 0.005 },
+      { "drop": "sweetbread", "type": "flesh", "mass_ratio": 0.002 },
+      { "drop": "kidney", "type": "offal", "mass_ratio": 0.002 },
+      { "drop": "stomach_large", "scale_num": [ 1, 1 ], "max": 1, "type": "offal" },
```
Comment (chaosvolt) [inline data/json/harvest.json:414]:
```suggestion
      { "drop": "brain", "type": "flesh", "mass_ratio": 0.005 },
      { "drop": "mutant_offal", "type": "offal", "mass_ratio": 0.01 },
      { "drop": "stomach_large", "scale_num": [ 1, 1 ], "max": 1, "type": "offal" },
```

**Review by chaosvolt — COMMENTED** [review]
I based these off `mammal_large_chitin`, which has brains and stomach but replaces all other organs with mutant offal at a ratio equal to twice the mass of the brains.

Increasing the mutant offal mass ratios a bit to account for there being fewer organ entries could be done if desired, though probably low priority since generic mutant offal is less useful recipe-wise than generic mutant meat, and probably less useful than any of the individual non-mutant organs either, making it less problematic that you get less of them from scaled bears anyway.

**Review by chaosvolt — APPROVED** [review]

**Pull request merged**